bot.py

```diff
@@ -154,6 +154,21 @@ def process_interactive():
         return make_response("", 200)
 
 
+@slack_events_adapter.on("channel_created")
+def channel_created(event_data):
+    user_id = event_data["event"]["channel"]["creator"]
+    channel_name = event_data["event"]["channel"]["id"]
```

Comment:
When using name it showed as private info in slack which is weird. id works fine but not very obvious.